services/tmdb/index.js

```diff
@@ -0,0 +1,29 @@
+const axios = require('axios');
+const { requests } = require('../../utils');
+
+exports.requestConfigurations = () => {
```

Comment:
This should be an implicit return, since that's all this function does.

```suggestion
const requestConfigurations = () => axios(requests.TMDB_CONFIG);
```